File: project-manager/src/main/java/uni/lodz/pl/projectmanager/sprint/model/AddSprintDto.java
package uni.lodz.pl.projectmanager.sprint.model;

import io.swagger.v3.oas.annotations.tags.Tag;
import lombok.Getter;
import lombok.Setter;

import java.time.LocalDate;

@Getter
@Setter
@Tag(name = "Add Sprint DTO")
public class AddSprintDto {
    private String name;
    private LocalDate start;
    private LocalDate end;
    private Long projectId;
}

File: project-manager-ui/src/model/sprint/AddSprintDto.ts
export class AddSprintDto {
  name: string;
  start: Date;
  end: Date;
  projectId: number;

  constructor(name: string, start: Date, end: Date, projectId: number) {
    this.name = name;
    this.start = start;
    this.end = end;
    this.projectId = projectId;
    return;
  }
}